etc/org.opencastproject.ingest.impl.IngestServiceImpl.cfg

```diff
@@ -6,9 +6,14 @@
 # In both cases, the series catalog will be created if it does not already exist in Opencast
 # In both cases, the catalog for the episode will be created in Opencast
 
-# TODO: consider moving this key to the system.properties if used by other services, such as mediapackage update
+# MH-12533 Option to prevent ingested flavors (except for tracks) to be overritten during ingest.
+# If true (default), ingested elements that match existing flavors take precedence and overwrite the existing.
+#    For example, metadata changes made from a capture agent UI will override metadata changes made from the Admin UI prior to the ingest.
+# If false, tracks and only new element flavors are accepted from the ingested mp. Existing flavors, like episode and series catalogs,
+#    are not overwritten and retain all updates made from the Admin UI.
+#    Changed the default from true to false to allow metadata edits from the Admin UI to take precedence.
 
-org.opencastproject.series.overwrite=true
+org.opencastproject.ingest.overwrite=true
```

Comment:
Can you use the following form:

```
# Some description
# Default: xy
#some.key=default.value
```

That way you can easily spot what the default is and you know right away that you did not modify the default since it's still commented out. This is used in a lot of other configuration files as well.

Reply:
@lkiesow thank you for this suggestion. It is changed to this format.